### PR TITLE
PB-897 Deployment work

### PR DESCRIPTION
**Pull request by schtibe**
Changing various stuff that's needed for service-control to run in k8s
* Missing / Wrong environment configurations
* Missing packages
* Missing wsgi file

### CODE REVIEW

File: app/wsgi.py [hunk 1]
```diff
@@ -1,16 +1,75 @@
...
+    import gevent.monkey
+    gevent.monkey.patch_all()
+"""
+WSGI config for project project.
 
 It exposes the WSGI callable as a module-level variable named ``application``.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/5.0/howto/deployment/wsgi/
+https://docs.djangoproject.com/en/3.1/howto/deployment/wsgi/
```
Comment (ltshb) [inline app/wsgi.py:28]: don't you want to point to django 5 here ?

**Review by ltshb — APPROVED** [review]
Good just a few minor comments

File: Pipfile [hunk 2]
```diff
@@ -4,10 +4,14 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-django = "~=5.0"
+django = "==5.0.7"
```
[the remark this answers is not in the document]
Comment (schtibe) [reply Pipfile:7]: Indeed, thanks

File: app/wsgi.py (hunk 1 above)
Comment (schtibe) [reply app/wsgi.py:28]: Oh sure, I overwrote the default file with our custom file, and thus copy pasted the ancient link.

**Pull request merged**